components/service/pocket/src/main/java/mozilla/components/service/pocket/spocs/SpocsUseCases.kt

```diff
@@ -7,53 +7,75 @@ package mozilla.components.service.pocket.spocs
 import android.content.Context
 import androidx.annotation.VisibleForTesting
 import mozilla.components.concept.fetch.Client
-import mozilla.components.service.pocket.PocketNetworkUseCase
-import mozilla.components.service.pocket.PocketNetworkUseCases
 import mozilla.components.service.pocket.PocketSponsoredStory
-import mozilla.components.service.pocket.logger
 import mozilla.components.service.pocket.spocs.api.SpocsEndpoint
-import mozilla.components.service.pocket.stories.api.PocketEndpoint
 import mozilla.components.service.pocket.stories.api.PocketResponse.Failure
 import mozilla.components.service.pocket.stories.api.PocketResponse.Success
 import java.util.UUID
 
 /**
  * Possible actions regarding the list of sponsored stories.
+ *
+ * @param appContext Android Context. Prefer sending application context to limit the possibility of even small leaks.
+ * @param fetchClient the HTTP client to use for network requests.
+ * @param profileId Unique profile identifier used for downloading sponsored Pocket stories.
+ * @param appId Unique app identifier used for downloading sponsored Pocket stories.
  */
-internal class SpocsUseCases : PocketNetworkUseCase() {
```

Comment:
Is`PocketNetworkUseCase` used anywhere else now?

Reply:
Thank you. This is what this patch meant to refactor out. 😅
Checked now. Not used anywhere else. Will remove this file.